schema_based_on_diagram.sql

```diff
@@ -0,0 +1,16 @@
+CREATE DATABASE schema_diagram;
+
+
+ CREATE TABLE patients(id INT PRIMARY KEY, name VARCHAR, date_of_birth DATE);
+
+ CREATE TABLE medical_histories(id INT PRIMARY KEY, admitted_at TIMESTAMP, patient_id INT, status VARCHAR, FOREIGN KEY (patient_id) REFERENCES patients(id));
+
+ CREATE TABLE treatments(id INT PRIMARY KEY, type VARCHAR, name VARCHAR);
```

Comment:
- [ ] From the schema diagram, we can see that the medical_histories table and the treatments table have a many-to-many relationship. This would require that you create a join table with foreign keys that reference the ids on the aforementioned tables as columns. Kindly create a join table for these two tables. 👍 😎 

#### Hint: You can call the table `medical_histories_has_treatments`. 💻